### PR TITLE
What's new for EF9 Preview 1

### PR DESCRIPTION
**Pull request by ajcvickers**

### CODE REVIEW

File: entity-framework/core/what-is-new/ef-core-9.0/whatsnew.md [hunk 1]
```diff
@@ -13,6 +13,598 @@ EF Core 9 (EF9) is the next release after EF Core 8 and is scheduled for release
...
 
 > [!TIP]
-> You can run and debug into the samples by [downloading the sample code from GitHub](https://github.com/dotnet/EntityFramework.Docs). Each section links to the source code specific to that section.
+> You can run and debug into the samples by [downloading the sample code from GitHub](https://github.com/dotnet/EntityFramework.Docs). Each section below links to the source code specific to that section.
+
+EF9 targets .NET 8, and can therefore be used with either [.NET 8 (LTS)](https://dotnet.microsoft.com/download/dotnet/8.0) or a .NET 9 preview.
+
+## What's new in EF Core 8 Preview 1
+
+The team has been working primarily on EF internals, so there are no new big features in EF Core 8 Preview 1. However, this means we need to get people like you (If you're reading a preview 1 post, then you're a really engaged part of the community; thank you!) to run your code on these new internals. We have over 120,000 tests, but it's not enough! We need you guys running your real code on our bits in order to find issues and ship a solid release!
```
Comment (AndriySvyryd) [inline entity-framework/core/what-is-new/ef-core-9.0/whatsnew.md:22]:
```suggestion
The team has been working primarily on EF internals, so there are no new big features in EF Core 8 Preview 1. However, this means we need to get people like you (If you're reading a preview 1 post, then you're a really engaged part of the community; thank you!) to run your code on these new internals. We have over 120,000 tests, but it's not enough! We need you, people running real code on our bits, in order to find issues and ship a solid release!
```

File: entity-framework/core/what-is-new/ef-core-9.0/whatsnew.md [hunk 2]
```diff
@@ -13,6 +13,598 @@ EF Core 9 (EF9) is the next release after EF Core 8 and is scheduled for release
 EF9 is available as [daily builds](https://github.com/dotnet/efcore/blob/main/docs/DailyBuilds.md) which contain all the latest EF9 features and API tweaks. The samples here make use of these daily builds.
 
 > [!TIP]
-> You can run and debug into the samples by [downloading the sample code from GitHub](https://github.com/dotnet/EntityFramework.Docs). Each section links to the source code specific to that section.
+> You can run and debug into the samples by [downloading the sample code from GitHub](https://github.com/dotnet/EntityFramework.Docs). Each section below links to the source code specific to that section.
+
+EF9 targets .NET 8, and can therefore be used with either [.NET 8 (LTS)](https://dotnet.microsoft.com/download/dotnet/8.0) or a .NET 9 preview.
+
+## What's new in EF Core 8 Preview 1
```
Comment (ErikEJ) [inline entity-framework/core/what-is-new/ef-core-9.0/whatsnew.md:20]: EF Core 9 ?

File: entity-framework/core/what-is-new/ef-core-9.0/whatsnew.md (hunk 1 above)
Comment (ErikEJ) [inline entity-framework/core/what-is-new/ef-core-9.0/whatsnew.md:22]: EF Core 9?

File: entity-framework/core/what-is-new/ef-core-9.0/whatsnew.md [hunk 3]
```diff
@@ -13,6 +13,598 @@ EF Core 9 (EF9) is the next release after EF Core 8 and is scheduled for release
...
+
+<!--
+        #region PruneJSON
+        var results = await context.Posts
+            .Where(p => p.Metadata!.Updates.Count(e => e.UpdatedOn >= date) == 1)
+            .ToListAsync();
+-->
+[!code-csharp[PruneJSON](../../../../samples/core/Miscellaneous/NewInEFCore9/JsonColumnsSample.cs?name=PruneJSON)]
+
+In EF8, this would query generates the following SQL when using the Azure SQL database provider:
```
Comment (AndriySvyryd) [inline entity-framework/core/what-is-new/ef-core-9.0/whatsnew.md:45]:
```suggestion
In EF8, this query generates the following SQL when using the Azure SQL database provider:
```

**Review by AndriySvyryd — APPROVED** [review]

File: entity-framework/core/what-is-new/ef-core-9.0/whatsnew.md (hunk 2 above)
Comment (ErikEJ) [inline entity-framework/core/what-is-new/ef-core-9.0/whatsnew.md:20]: Should it not be "What's new in EF Core 9 Preview 1" ?

**Pull request merged**